blobmodel/model.py

```diff
@@ -21,7 +22,7 @@ def __init__(
         dt: float = 0.1,
         T: float = 10,
         periodic_y: bool = False,
-        blob_shape: str = "gauss",
+        blob_shape: Union[AbstractBlobShape, str] = BlobShapeImpl("gauss"),
```

Comment:
I don't like the fact that one needs to import another class (`BlobShapeImpl`) in order to instantiate a `model` object. Also, that breaks the API so we would have to increase the version number. Could we keep `blob_shape` as a string here and move instantiation of `BlobShapeImpl` into the constructor?

Reply:
That's the reason why I allow blob_shape to be a string. I think it doesn't break the API this way, right?